src/hotspot/share/prims/scopedMemoryAccess.cpp

```diff
@@ -160,13 +160,12 @@ JVM_END
 
 #define MEMACCESS "ScopedMemoryAccess"
 #define SCOPE PKG_FOREIGN "ResourceScope;"
-#define SCOPED_ERR PKG_MISC MEMACCESS "$ScopedAccessError;"
 
 #define CC (char*)  /*cast a literal from (const char*)*/
 #define FN_PTR(f) CAST_FROM_FN_PTR(void*, &f)
 
 static JNINativeMethod jdk_internal_misc_ScopedMemoryAccess_methods[] = {
-    {CC "closeScope0",   CC "(" SCOPE SCOPED_ERR ")Z",           FN_PTR(ScopedMemoryAccess_closeScope)},
+    {CC "closeScope0",   CC "(" SCOPE ")Z",           FN_PTR(ScopedMemoryAccess_closeScope)},
```

Comment:
Uhmm - it seems like the method declaration for `ScopedMemoryAccess_closeScope` still has a trailing exception object (further down in this file)

Reply:
Oops, thanks for spotting. That's removed now.